third_party/terraform/resources/resource_sql_database_instance.go

```diff
@@ -452,6 +451,13 @@ func isFirstGen(d *schema.ResourceData) bool {
 	return !regexp.MustCompile("db*").Match([]byte(tier))
 }
 
+func isSqlOperationInProgressError(err error) (bool, string) {
+	if gerr, ok := err.(*googleapi.Error); ok && gerr.Code == 409 {
+		return true, "" //"fmt.Errorf("Error, failed to create instance %s with error code 409: %s. This may be due to a name collision - SQL instance names cannot be reused within a week.", instance.Name, err)"
```

Comment:
Do you mind deleting this comment?

Reply:
oh sorry, I was going to figure out what string I wanted there, will fix it